src/app/services/auth-manager.service.ts

```diff
@@ -1,17 +1,60 @@
-import {Injectable} from '@angular/core';
-import {LocalStorageWrapper} from './local-storage-wrapper';
+import {Injectable} from "@angular/core";
+import {ApiCall} from "./api-call";
+import "rxjs/add/operator/map";
```

Comment:
Extract to 'rxjs-extensions.ts' with the used imports.
Import it to app.module.ts